bears/general/InvalidLinkBear.py

```diff
@@ -33,14 +33,20 @@ def find_links_in_file(file):
             match = regex.search(line)
             if match:
                 link = match.group()
-                code = InvalidLinkBear.get_status_code_or_error(link)
+                code = InvalidLinkBear.get_status_code_or_error(link, timeout)
                 yield line_number + 1, link, code
 
-    def run(self, filename, file):
+    def run(self, filename, file, timeout: str):
```

Comment:
Alright. I didn't understand that at first. I'll change the data type to `int` and directly set `timeout: int=2` in the param.
Correct?